active/NumberBox/NumberBox.md

```diff
@@ -159,24 +185,31 @@ unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.Control
     String Text; 
     String PlaceholderText;
 
+<!-- todo: Why does this say "basic"? Is there also an advanced validation mode? -->   
     NumberBoxBasicValidationMode BasicValidationMode;
 
     Boolean AcceptsCalculation;
 
     NumberBoxSpinButtonPlacementMode SpinButtonPlacementMode{ get; set; };
 
+<!-- todo: This should have the "Is" prefix, to differentiate them from events. -->
     Boolean HyperScrollEnabled;
     Boolean HyperDragEnabled;
```

Comment:
This doesn't exist yet and sounds like it probably won't for v1.

Reply:
I have added the "Is" prefix to all hyper and wrap properties. 

Hyper properties can be re-prioritized for the 3.0.